container/root/scripts/install_goss.sh

```diff
@@ -6,14 +6,11 @@
 # Downloads, verifies, and installs
 # Requires curl and sha256sum to be present
 
-GOSS_VERSION=v0.3.8
-GOSS_SHA256=4c82470543350371531f26f9d3b0265bff9e93d80d952f40cca212fc2f87864d
+GOSS_VERSION=v0.3.9
+GOSS_SHA256=5e4a51a8c0f955e5ce99851b4a15eed9f1b3b6bee17f23dabda08071775663c8
 
 curl -fL https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 -o /usr/local/bin/goss
 
-curl -fL https://github.com/aelsabbahy/goss/releases/download/v0.3.8/goss-linux-amd64 -o /usr/local/bin/goss
```

Comment:
duplicative, looks accidental and incorrect

Reply:
👍